.github/workflows/release.yml

```
@@ -64,6 +64,8 @@ jobs:
           path: share
       - run: tar cvzf ${{ inputs.project }}.${{ inputs.target }}.tgz ${{ inputs.project }}* share
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
+      - run: zip -R ${{ inputs.project }}.${{ inputs.target }}.zip ${{ inputs.project }}* share
+        if: ${{ startsWith(inputs.target, 'x86_64-windows') }}
```

```suggestion
        if: ${{ startsWith(github.ref, 'refs/tags/') && endsWith(inputs.target, '-windows') }}
```

1. It archives only when a git tag is present (release)
2. It covers any future Windows architecture